packages/blockchain-utils-linking/src/tezos.ts

```diff
@@ -0,0 +1,117 @@
+import { AccountID } from 'caip'
+import { AuthProvider } from './auth-provider'
+import { getConsentMessage, LinkProof } from './util'
+import { hash } from '@stablelib/sha256'
+import * as uint8arrays from 'uint8arrays'
+import { SetProviderOptions, TezosToolkit } from '@taquito/taquito'
+import { char2Bytes } from '@taquito/utils'
+
+export type TezosProvider = SetProviderOptions
+
+/**
+ * initializes the TezosToolkit with the given {TezosProvider} and returns the {TezosToolkit}
+ *
+ * @param {TezosProvider} provider - the provider to use for initializing the TezosToolkit
+ * @returns {TezosToolkit} the initialized TezosToolkit
+ */
+function getTezosToolkit(provider: TezosProvider) {
+  // initialize the Tezos RPC client
+  const Tezos = new TezosToolkit('https://mainnet-tezos.giganode.io')
```

Comment:
@Geo25rey Errm, are you sure it is a good idea to have depend on this HTTP endpoint??

Reply:
It's a required argument. I don't think it's used in this context tho. This should be getting removed anyway. See https://github.com/ceramicnetwork/js-ceramic/pull/1555#discussion_r671802423